virt/virt-4-8-release-notes.adoc

```diff
@@ -0,0 +1,150 @@
+[id="virt-4-8-release-notes"]
+= {RN_BookName}
+include::modules/virt-document-attributes.adoc[]
+include::modules/common-attributes.adoc[]
+:context: virt-release-notes
+
+toc::[]
+
+== About Red Hat {VirtProductName}
+
+Red Hat {VirtProductName} enables you to bring traditional virtual machines (VMs) into {product-title} where they run alongside containers, and are managed as native Kubernetes objects.
+
+{VirtProductName} is represented by the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName},40,40] logo.
+
+You can use {VirtProductName} with either the xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes] or the xref:../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShiftSDN] default Container Network Interface (CNI) network provider.
+
+Learn more about xref:../virt/about-virt.adoc#about-virt[what you can do with {VirtProductName}].
+
+include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
+
+[id="virt-guest-os"]
+=== Supported guest operating systems
+{VirtProductName} guests can use the following operating systems:
+
+* Red Hat Enterprise Linux 6, 7, and 8.
+* Microsoft Windows Server 2012 R2, 2016, and 2019.
+* Microsoft Windows 10.
+
+Other operating system templates shipped with {VirtProductName} are not supported.
+
+[id="virt-4-8-inclusive-language"]
+== Making open source more inclusive
+
+Red Hat is committed to replacing problematic language in our code, documentation, and web properties. We are beginning with these four terms: master, slave, blacklist, and whitelist. Because of the enormity of this endeavor, these changes will be implemented gradually over several upcoming releases. For more details, see link:https://www.redhat.com/en/blog/making-open-source-more-inclusive-eradicating-problematic-language[our CTO Chris Wright's message].
+
+[id="virt-4-8-new"]
+== New and changed features
+
+* OpenShift Virtualization is certified in Microsoft’s Windows Server Virtualization Validation Program (SVVP) to run Windows Server workloads.
++
+The SVVP Certification applies to:
++
+** Red Hat Enterprise Linux CoreOS workers. In the Microsoft SVVP Catalog, they are named __Red Hat OpenShift Container Platform 4 on RHEL CoreOS__.
+** Intel and AMD CPUs.
+
+
+[id="virt-4-8-installation-new"]
+=== Installation
+
+//CNV-8167 Supported guest operating systems under Openshift virtualization 2.4
```

Comment:
This story is a recurring release note and is part of the `Supported guest operating systems` section above